### DISCUSSION TITLE
.PNG transparency support

### DISCUSSION THREAD

**Discussion by VegaDeftwing** [noisedeck/noisedeck-docs#20] [Ideas]
It would be nice if pixy fully supported .png transparency. As is, fully transparent regions are made transparent, but translucent regions are set to fully opaque. It would also be nice if the background color for where the image doesn't cover could be set

**Comment by jesshewitt** [comment]
Pixy does support transparency, but only if you have a second synth module enabled. You could use Quadruple Tap as the second synth to set a solid background color.

**Reply by VegaDeftwing** [reply]
Using a quad tap to set the background color works, but it's a bit annoying to have to use the second synth module on that. ¯\\_(ツ)_/¯

**Reply by jesshewitt** [reply]
I've added the background color option to pixy - it will be in the next release.
